addons/sourcemod/scripting/gokz-core/forwards.sp

```diff
@@ -82,6 +82,7 @@ void CreateGlobalForwards()
 	H_OnOptionsMenuReady = new GlobalForward("GOKZ_OnOptionsMenuReady", ET_Ignore, Param_Cell);
 	H_OnCourseRegistered = new GlobalForward("GOKZ_OnCourseRegistered", ET_Ignore, Param_Cell);
 	H_OnRunInvalidated = new GlobalForward("GOKZ_OnRunInvalidated", ET_Ignore, Param_Cell);
+	H_OnEmitSoundToClient = new GlobalForward("GOKZ_OnEmitSoundToClient", ET_Hook, Param_Cell, Param_String, Param_FloatByRef);
```

Comment:
This is missing another string param for `description` if I'm not mistaken.

Reply:
Ah right, I forgot this part.